ludwig/backend/ray.py

```diff
@@ -251,6 +327,49 @@ def train(
     def train_online(self, *args, **kwargs):
         raise NotImplementedError()
 
+    def tune_batch_size(
+        self,
+        config: Dict[str, Any],
+        training_set: RayDataset,
+        **kwargs,
+    ) -> int:
+        trainer_kwargs = {**get_trainer_kwargs(), **self.trainer_kwargs}
```

Comment:
nit: We run this code snippet of setting CPU and GPU resources at a few other places inside the ray backend, so it would be good to abstract this out into a function.